src/main/java/core/basesyntax/Default.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Default implements Figure {
```

Comment:
The class name 'Default' is a verb, which is not recommended for class names. Consider using a noun that better describes the purpose of this class.